uninstall.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * performance-lab uninstall method
+ */
+
+// if uninstall.php is not called by WordPress, die
```

Comment:
Comments should start with a capital letter and end in a full stop.
```suggestion
// If uninstall.php is not called by WordPress, bail.
```

Reply:
Resolved, thank you